packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -215,27 +77,50 @@ class ConfigurationPanel extends React.Component<
                         onChange: this.handleLocationUpdate,
                     }}
                 />
-                {this.renderCSSEditor()}
             </React.Fragment>
         );
     }
 
     private renderCSSEditor(): React.ReactNode {
         if (!!this.props.styleEditing) {
-            return <CSSEditor onChange={this.handleStyleChange} data={this.getStyle()} />;
+            return (
+                <React.Fragment>
+                    {this.renderCategoryTitle("CSS")}
+                    <CSSEditor
+                        jssStyleSheet={{ cssEditor: { height: "unset" } }}
+                        onChange={this.handleStyleChange}
+                        data={this.getStyle()}
+                    />
+                </React.Fragment>
+            );
         }
 
         return null;
     }
 
-    private getTabClassNames(tabItem: TabType): string {
-        if (tabItem === this.state.activeTab) {
-            return `${this.props.managedClasses.configurationPanel_tab} ${
-                this.props.managedClasses.configurationPanel_tab__active
-            }`;
+    private renderCSSPropertyEditor(): React.ReactNode {
+        if (!!this.props.styleEditing) {
+            return (
+                <div style={{ padding: "0 10px" }}>
```

Comment:
Is there a reason you aren't applying styles via classes and doing it inline instead?

Reply:
Because it's a single property, not based on the design system. I don't see measurable a benefit of doing it one way over another.

Reply:
Having said that, I can convert it if others feel strongly about it.

Reply:
Got it. I definitely do not have strong feelings - I was just curious.

Reply:
I took action on it anyway.